requestSignatureVerification property added to Application Page v1.0 (#19894)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/requestsignatureverification.md

---
title: "requestSignatureVerification resource type"
description: "Specifies whether this application requires Azure AD to verify the signed authentication requests."
author: "rahul-nagraj"
ms.localizationpriority: medium
ms.prod: "applications"
doc_type: resourcePageType
---

# requestSignatureVerification resource type

Namespace: microsoft.graph

Specifies whether this application requires Azure AD to verify the signed authentication requests.

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |allowedWeakAlgorithms|weakAlgorithms|Specifies whether this application accepts weak algorithms. <br><br> The possible values are: `rsaSha1`, `unknownFutureValue`.|

Reviewer: The description of the property can be interpreted as a boolean type.  Also, consider updating the api.md file for future references as it has a difference in properties defined.
```suggestion
|allowedWeakAlgorithms|weakAlgorithms|Specifies which weak algorithms are allowed. <br><br> The possible values are: `rsaSha1`, `unknownFutureValue`.|
```
